tools/docker/docker-compose-dev.yaml

```diff
@@ -199,4 +199,4 @@ volumes:
 
 networks:
   service-mesh:
-    name: service-mesh
+    name: service-mesh
```

Comment:
I think you don't have rebased it properly, this change is already on main. https://github.com/ansible/eda-server/blob/main/tools/docker/docker-compose-dev.yaml#L202

Reply:
have you look at this?